src/components/header/renderer.tsx

```diff
@@ -170,20 +179,21 @@ class Header extends React.Component<Props, State> {
       }
     }
 
-    this.props.history.push(`/gist/${type}/${username}/${gistId}/${revision}/${filename}`);
-
-    this.setState({
-      gist: {
-        filename: '',
-        revision: '',
-        type: Mode.Vega,
-        url: '',
-      },
+    if (!this.state.invalidUrl) {
+      this.props.history.push(`/gist/${type}/${username}/${gistId}/${revision}/${filename}`);
+      this.setState({
+        gist: {
+          filename: '',
+          revision: '',
+          type: Mode.Vega,
+          url: '',
+        },
 
-      invalidUrl: false,
-    });
+        invalidUrl: false,
+      });
 
-    closePortal(); // Close the gist modal after it gets load
+      closePortal(); // Close the gist modal after it gets load
+    }
```

Comment:
Else? How does the user get feedback? Shouldn’t we show an error message?

Reply:
Nm, found it

Reply:
There is an error message
![corrected7](https://user-images.githubusercontent.com/35253743/53742969-d1243f00-3ebf-11e9-8481-f11b4794e684.png)
The message was already there before the pr but it wasn't being displayed.
Now it gets displayed after the load button is hit.

Reply:
Thanks